examples/keras_io/tensorflow/nlp/multi_label_classification.py

```diff
@@ -0,0 +1,386 @@
+"""
+Title: Large-scale multi-label text classification
+Author: [Sayak Paul](https://twitter.com/RisingSayak), [Soumik Rakshit](https://github.com/soumik12345)
+Converted to KerasCore by: [Akorede Adewole](https://github.com/Akorex)
+Date created: 2020/09/25
+Last modified: 2023/7/19
+Description: Implementing a large-scale multi-label text classification model.
+
+"""
+
+
+"""
+## Introduction
+
+In this example, we will build a multi-label text classifier to predict the subject areas
+of arXiv papers from their abstract bodies. This type of classifier can be useful for
+conference submission portals like [OpenReview](https://openreview.net/). Given a paper
+abstract, the portal could provide suggestions for which areas the paper would
+best belong to.
+
+The dataset was collected using the
+[`arXiv` Python library](https://github.com/lukasschwab/arxiv.py)
+that provides a wrapper around the
+[original arXiv API](http://arxiv.org/help/api/index).
+To learn more about the data collection process, please refer to
+[this notebook](https://github.com/soumik12345/multi-label-text-classification/blob/master/arxiv_scrape.ipynb).
+Additionally, you can also find the dataset on
+[Kaggle](https://www.kaggle.com/spsayakpaul/arxiv-paper-abstracts).
+
+
+## Imports
+"""
+
+
+!pip install -q git+https://github.com/keras-team/keras-core
+
+import keras_core as keras
+from keras_core import layers
+import tensorflow as tf
+
+from sklearn.model_selection import train_test_split
```

Comment:
You can use `split_dataset` for this.

Reply:
Yes, but the original example stratified the split on a column because of class imbalance. `split_dataset` to my knowledge isn't supported for this.

Reply:
Yes, but the original example stratified the split on a column because of class imbalance. `split_dataset` to my knowledge isn't supported for this.

Reply:
Ok